kura/examples/org.eclipse.kura.raspberrypi.sensehat.example/resources/dp/org.eclipse.kura.raspberrypi.sensehat.example.dpp

```diff
@@ -0,0 +1,27 @@
+#Deployment Plugin Project File
```

Comment:
@pierantoniomerlino Do we want to push dps and dpps? (We have also references to your filesystem) Maybe it's better to modify the distrib pom file to create the corresponding dp file during build